"use strict";(self["webpackChunkquantms_org"]=self["webpackChunkquantms_org"]||[]).push([[557],{1605:(e,t,a)=>{a(6422),a(9267)},2261:(e,t,a)=>{a(6422)},5283:(e,t,a)=>{a.d(t,{a:()=>c});var i=a(641),l=a(5220);const n={title:"quantms - Cloud-based Pipeline for Quantitative Proteomics",description:"quantms is a bioinformatics best-practice analysis pipeline for Quantitative Mass Spectrometry (MS), including DDA-LFQ, plexDDA (TMT, iTRAQ), and DIA.",keywords:"quantms, proteomics, mass spectrometry, bioinformatics, DDA-LFQ, TMT, iTRAQ, DIA, OpenMS",author:"Yasset Perez-Riverol, Timo Sachsenberg, Mingze Bai, Chengxin Dai",image:"https://quantms.org/logo.png"},s={"/":n,"/publications":{title:"Publications - quantms",description:"Scientific publications related to quantms, including research papers using the quantms pipeline and its components.",keywords:"quantms publications, proteomics research, mass spectrometry papers, scientific publications"},"/datasets":{title:"Datasets - quantms",description:"Browse and access proteomics datasets reanalyzed using the quantms pipeline, including absolute expression, differential expression, and single-cell data.",keywords:"proteomics datasets, mass spectrometry data, reanalyzed data, expression data"},"/about":{title:"About - quantms",description:"Learn about the quantms team, contributors, and the development of this cloud-based pipeline for quantitative proteomics.",keywords:"quantms team, proteomics pipeline, development team, contributors"}},o=(e,t)=>{let a=document.querySelector(`meta[name="${e}"]`)||document.querySelector(`meta[property="${e}"]`);a||(a=document.createElement("meta"),a.setAttribute(e.startsWith("og:")?"property":"name",e),document.head.appendChild(a)),a.setAttribute("content",t)},r=()=>{const e=document.createElement("script");e.setAttribute("type","application/ld+json");const t={"@context":"https://schema.org","@type":"WebPage",name:"quantms Publications",description:"Scientific publications related to quantms and its applications in proteomics",mainEntity:{"@type":"ItemList",itemListElement:[{"@type":"ScholarlyArticle",headline:"quantms: a cloud-based pipeline for quantitative proteomics enables the reanalysis of public proteomics data",author:[{"@type":"Person",name:"Chengxin Dai"},{"@type":"Person",name:"Yasset Perez-Riverol"}],datePublished:"2024-09",publisher:{"@type":"Organization",name:"Nature Methods"},isAccessibleForFree:!0}]}};e.textContent=JSON.stringify(t),document.head.appendChild(e)};function c(){const e=(0,l.lq)();(0,i.sV)((()=>{const t=s[e.path]||n;document.title=t.title,o("description",t.description),o("keywords",t.keywords),o("author",t.author||n.author),o("og:title",t.title),o("og:description",t.description),o("og:image",t.image||n.image),o("og:url",`https://quantms.org${e.path}`),o("og:type","website"),o("twitter:card","summary_large_image"),o("twitter:title",t.title),o("twitter:description",t.description),o("twitter:image",t.image||n.image),"/publications"===e.path&&r()}))}},6262:(e,t)=>{t.A=(e,t)=>{const a=e.__vccOpts||e;for(const[i,l]of t)a[i]=l;return a}},6422:()=>{},9267:(e,t,a)=>{a(6422)},9303:(e,t,a)=>{a(6422)},9557:(e,t,a)=>{a.r(t),a.d(t,{default:()=>U});var i=a(6796),l=(a(6422),a(2261),a(9999),a(9178)),n=(a(1605),a(1857)),s=(a(9303),a(641)),o=a(5283);const r={class:"publications-page"},c={class:"publications-section"},u={class:"publication-category container"},p={class:"publication-card"},d={class:"publication-links"},m={href:"https://doi.org/10.1038/s41592-024-02343-1",target:"_blank",class:"publication-link"},k={class:"publication-category container"},b={class:"publications-table-container"},h={class:"publications-table"},g={class:"link-column"},L={href:"https://doi.org/10.1016/j.jprot.2025.105440",target:"_blank",class:"publication-link"},f={class:"citation-text"},y={class:"link-column"},P={href:"https://doi.org/10.1016/j.molcel.2024.02.011",target:"_blank",class:"publication-link"},v={class:"link-column"},_={href:"https://doi.org/10.1038/s41592-024-02197-7",target:"_blank",class:"publication-link"},M={class:"link-column"},W={href:"https://doi.org/10.1002/pmic.202300188",target:"_blank",class:"publication-link"},D={class:"link-column"},C={href:"https://doi.org/10.1021/acs.jproteome.2c00812",target:"_blank",class:"publication-link"},S={class:"link-column"},q={href:"https://doi.org/10.1093/bioinformatics/btab838",target:"_blank",class:"publication-link"},z={class:"link-column"},A={href:"https://doi.org/10.1038/s41467-021-26111-3",target:"_blank",class:"publication-link"},w={class:"link-column"},I={href:"https://doi.org/10.1101/2024.05.24.595489",target:"_blank",class:"publication-link"},F={class:"publications-table-container"},B={class:"publications-table"},x={class:"citation-text"},T={class:"link-column"},J={href:"https://doi.org/10.1089/neu.2023.0064",target:"_blank",class:"publication-link"},R={class:"link-column"},E={href:"https://doi.org/10.1016/j.envpol.2025.126386",target:"_blank",class:"publication-link"},V={class:"link-column"},j={href:"https://doi.org/10.1016/j.envpol.2025.126214",target:"_blank",class:"publication-link"},N={class:"citation-text"},K={class:"link-column"},H={href:"https://doi.org/10.1016/j.molcel.2024.02.011",target:"_blank",class:"publication-link"},O={class:"link-column"},Y={href:"https://doi.org/10.1021/acs.jproteome.2c00607",target:"_blank",class:"publication-link"},G={__name:"index",setup(e){return(0,o.a)(),(e,t)=>{const a=n.S2,o=l.R7,G=i.v$,Q=i.q;return(0,s.uX)(),(0,s.CE)("div",r,[(0,s.Lk)("section",c,[t[47]||(t[47]=(0,s.Lk)("h1",{class:"section-title"},"Publications",-1)),(0,s.Lk)("div",u,[t[5]||(t[5]=(0,s.Lk)("h2",{class:"category-title"},"Main Publication",-1)),(0,s.Lk)("div",p,[t[1]||(t[1]=(0,s.Lk)("h3",{class:"publication-title"}," quantms: a cloud-based pipeline for quantitative proteomics enables the reanalysis of public proteomics data ",-1)),t[2]||(t[2]=(0,s.Lk)("p",{class:"publication-authors"}," Dai C, Pfeuffer J, Wang H, Zheng P, Käll L, Sachsenberg T, Demichev V, Bai M, Kohlbacher O, Perez-Riverol Y. ",-1)),t[3]||(t[3]=(0,s.Lk)("p",{class:"publication-journal"}," Nature Methods. 2024 Sep;21(9):1603-1607. ",-1)),t[4]||(t[4]=(0,s.Lk)("p",{class:"publication-details"}," doi: 10.1038/s41592-024-02343-1. Epub 2024 Jul 4. PMID: 38965444; PMCID: PMC11399091. ",-1)),(0,s.Lk)("div",d,[(0,s.Lk)("a",m,[(0,s.bF)(a,{type:"primary",size:"small"},{default:(0,s.k6)((()=>t[0]||(t[0]=[(0,s.eW)("View Publication")]))),_:1,__:[0]})])])])]),(0,s.Lk)("div",k,[t[46]||(t[46]=(0,s.Lk)("h2",{class:"category-title"},"Publications",-1)),(0,s.bF)(Q,{type:"border-card",class:"publication-tabs"},{default:(0,s.k6)((()=>[(0,s.bF)(G,{label:"Team Publications (7)"},{default:(0,s.k6)((()=>[(0,s.Lk)("div",b,[(0,s.Lk)("table",h,[t[26]||(t[26]=(0,s.Lk)("thead",null,[(0,s.Lk)("tr",null,[(0,s.Lk)("th",null,"Citation"),(0,s.Lk)("th",{class:"link-column"},"Link")])],-1)),(0,s.Lk)("tbody",null,[(0,s.Lk)("tr",null,[t[7]||(t[7]=(0,s.Lk)("td",null,[(0,s.Lk)("p",{class:"citation-text"},[(0,s.eW)(" Zheng P, Audain E, Webel H, Dai C, Klein J, Hitz MP, Sachsenberg T, Bai M, Perez-Riverol Y. (2025). Ibaqpy: A scalable Python package for baseline quantification in proteomics leveraging SDRF metadata. "),(0,s.Lk)("em",null,"Journal of Proteomics"),(0,s.eW)(", 317:105440. doi: 10.1016/j.jprot.2025.105440. PMID: 40268243. ")])],-1)),(0,s.Lk)("td",g,[(0,s.Lk)("a",L,[(0,s.bF)(a,{type:"primary",size:"small"},{default:(0,s.k6)((()=>t[6]||(t[6]=[(0,s.eW)("View")]))),_:1,__:[6]})])])]),(0,s.Lk)("tr",null,[(0,s.Lk)("td",null,[(0,s.Lk)("p",f,[t[9]||(t[9]=(0,s.eW)(" Arecco N, Mocavini I, Blanco E, Ballaré C, Libman E, Bonnal S, Irimia M, Di Croce L. (2024). Alternative splicing decouples local from global PRC2 activity. ")),t[10]||(t[10]=(0,s.Lk)("em",null,"Molecular Cell",-1)),t[11]||(t[11]=(0,s.eW)(", 84(6):1049-1061.e8. doi: 10.1016/j.molcel.2024.02.011. PMID: 38452766 ")),(0,s.bF)(o,{content:"This manuscript uses the proteomicslfq pipeline, an early version of quantms",placement:"top",effect:"light"},{default:(0,s.k6)((()=>t[8]||(t[8]=[(0,s.Lk)("span",{class:"tooltip-trigger"},"*",-1)]))),_:1,__:[8]}),t[12]||(t[12]=(0,s.eW)(" . "))])]),(0,s.Lk)("td",y,[(0,s.Lk)("a",P,[(0,s.bF)(a,{type:"primary",size:"small"},{default:(0,s.k6)((()=>t[13]||(t[13]=[(0,s.eW)("View")]))),_:1,__:[13]})])])]),(0,s.Lk)("tr",null,[t[15]||(t[15]=(0,s.Lk)("td",null,[(0,s.Lk)("p",{class:"citation-text"},[(0,s.eW)(" Pfeuffer J, Bielow C, Wein S, Jeong K, Netz E, Walter A, Alka O, Nilse L, Colaianni PD, McCloskey D, Kim J, Rosenberger G, Bichmann L, Walzer M, Veit J, Boudaud B, Bernt M, Patikas N, Pilz M, Startek MP, Kutuzova S, Heumos L, Charkow J, Sing JC, Feroz A, Siraj A, Weisser H, Dijkstra TMH, Perez-Riverol Y, Röst H, Kohlbacher O, Sachsenberg T. (2024). OpenMS 3 enables reproducible analysis of large-scale mass spectrometry data. "),(0,s.Lk)("em",null,"Nature Methods"),(0,s.eW)(", 21(3):365-367. doi: 10.1038/s41592-024-02197-7. PMID: 38366242. ")])],-1)),(0,s.Lk)("td",v,[(0,s.Lk)("a",_,[(0,s.bF)(a,{type:"primary",size:"small"},{default:(0,s.k6)((()=>t[14]||(t[14]=[(0,s.eW)("View")]))),_:1,__:[14]})])])]),(0,s.Lk)("tr",null,[t[17]||(t[17]=(0,s.Lk)("td",null,[(0,s.Lk)("p",{class:"citation-text"},[(0,s.eW)(" Wang H, Dai C, Pfeuffer J, Sachsenberg T, Sanchez A, Bai M, Perez-Riverol Y. (2023). Tissue-based absolute quantification using large-scale TMT and LFQ experiments. "),(0,s.Lk)("em",null,"Proteomics"),(0,s.eW)(", 23(20):e2300188. doi: 10.1002/pmic.202300188. PMID: 37488995. ")])],-1)),(0,s.Lk)("td",M,[(0,s.Lk)("a",W,[(0,s.bF)(a,{type:"primary",size:"small"},{default:(0,s.k6)((()=>t[16]||(t[16]=[(0,s.eW)("View")]))),_:1,__:[16]})])])]),(0,s.Lk)("tr",null,[t[19]||(t[19]=(0,s.Lk)("td",null,[(0,s.Lk)("p",{class:"citation-text"},[(0,s.eW)(" Bai M, Deng J, Dai C, Pfeuffer J, Sachsenberg T, Perez-Riverol Y. (2023). LFQ-Based Peptide and Protein Intensity Differential Expression Analysis. "),(0,s.Lk)("em",null,"Journal of Proteome Research"),(0,s.eW)(", 22(6):2114-2123. doi: 10.1021/acs.jproteome.2c00812. PMID: 37220883; PMCID: PMC10243145. ")])],-1)),(0,s.Lk)("td",D,[(0,s.Lk)("a",C,[(0,s.bF)(a,{type:"primary",size:"small"},{default:(0,s.k6)((()=>t[18]||(t[18]=[(0,s.eW)("View")]))),_:1,__:[18]})])])]),(0,s.Lk)("tr",null,[t[21]||(t[21]=(0,s.Lk)("td",null,[(0,s.Lk)("p",{class:"citation-text"},[(0,s.eW)(" Umer HM, Audain E, Zhu Y, Pfeuffer J, Sachsenberg T, Lehtiö J, Branca RM, Perez-Riverol Y. (2022). Generation of ENSEMBL-based proteogenomics databases boosts the identification of non-canonical peptides. "),(0,s.Lk)("em",null,"Bioinformatics"),(0,s.eW)(", 38(5):1470-1472. doi: 10.1093/bioinformatics/btab838. PMID: 34904638; PMCID: PMC8825679. ")])],-1)),(0,s.Lk)("td",S,[(0,s.Lk)("a",q,[(0,s.bF)(a,{type:"primary",size:"small"},{default:(0,s.k6)((()=>t[20]||(t[20]=[(0,s.eW)("View")]))),_:1,__:[20]})])])]),(0,s.Lk)("tr",null,[t[23]||(t[23]=(0,s.Lk)("td",null,[(0,s.Lk)("p",{class:"citation-text"},[(0,s.eW)(" Dai C, Füllgrabe A, Pfeuffer J, Solovyeva EM, Deng J, Moreno P, Kamatchinathan S, Kundu DJ, George N, Fexova S, Grüning B, Föll MC, Griss J, Vaudel M, Audain E, Locard-Paulet M, Turewicz M, Eisenacher M, Uszkoreit J, Van Den Bossche T, Schwämmle V, Webel H, Schulze S, Bouyssié D, Jayaram S, Duggineni VK, Samaras P, Wilhelm M, Choi M, Wang M, Kohlbacher O, Brazma A, Papatheodorou I, Bandeira N, Deutsch EW, Vizcaíno JA, Bai M, Sachsenberg T, Levitsky LI, Perez-Riverol Y. (2021). A proteomics sample metadata representation for multiomics integration and big data analysis. "),(0,s.Lk)("em",null,"Nature Communications"),(0,s.eW)(", 12(1):5854. doi: 10.1038/s41467-021-26111-3. PMID: 34615866; PMCID: PMC8494749. ")])],-1)),(0,s.Lk)("td",z,[(0,s.Lk)("a",A,[(0,s.bF)(a,{type:"primary",size:"small"},{default:(0,s.k6)((()=>t[22]||(t[22]=[(0,s.eW)("View")]))),_:1,__:[22]})])])]),(0,s.Lk)("tr",null,[t[25]||(t[25]=(0,s.Lk)("td",null,[(0,s.Lk)("p",{class:"citation-text"},[(0,s.eW)(" Wang D, Bouwmeester R, Zheng P, Dai C, Sanchez A, Shu K, Bai M, Umer HM, Perez-Riverol Y. (2024). Proteogenomics analysis of human tissues using pangenomes. "),(0,s.Lk)("em",null,"bioRxiv"),(0,s.eW)(" 2024.05.24.595489. doi: 10.1101/2024.05.24.595489. ")])],-1)),(0,s.Lk)("td",w,[(0,s.Lk)("a",I,[(0,s.bF)(a,{type:"primary",size:"small"},{default:(0,s.k6)((()=>t[24]||(t[24]=[(0,s.eW)("View")]))),_:1,__:[24]})])])])])])])])),_:1}),(0,s.bF)(G,{label:"Using quantms (5)"},{default:(0,s.k6)((()=>[(0,s.Lk)("div",F,[(0,s.Lk)("table",B,[t[45]||(t[45]=(0,s.Lk)("thead",null,[(0,s.Lk)("tr",null,[(0,s.Lk)("th",null,"Citation"),(0,s.Lk)("th",{class:"link-column"},"Link")])],-1)),(0,s.Lk)("tbody",null,[(0,s.Lk)("tr",null,[(0,s.Lk)("td",null,[(0,s.Lk)("p",x,[t[28]||(t[28]=(0,s.eW)(" Bernardo Harrington GM, Cool P, Hulme C, Fisher-Stokes J, Peffers M, Masri WE, Osman A, Chowdhury JR, Kumar N, Budithi S, Wright K. (2025). A Comprehensive Proteomic and Bioinformatic Analysis of Human Spinal Cord Injury Plasma Identifies Proteins Associated with the Complement Cascade and Liver Function as Potential Prognostic Indicators of Neurological Outcome. ")),t[29]||(t[29]=(0,s.Lk)("em",null,"Journal of Neurotrauma",-1)),t[30]||(t[30]=(0,s.eW)(", 42(3-4):292-306. doi: 10.1089/neu.2023.0064. PMID: 39636693 ")),(0,s.bF)(o,{content:"This manuscript uses the proteomicslfq pipeline, an early version of quantms",placement:"top",effect:"light"},{default:(0,s.k6)((()=>t[27]||(t[27]=[(0,s.Lk)("span",{class:"tooltip-trigger"},"*",-1)]))),_:1,__:[27]}),t[31]||(t[31]=(0,s.eW)(" . "))])]),(0,s.Lk)("td",T,[(0,s.Lk)("a",J,[(0,s.bF)(a,{type:"primary",size:"small"},{default:(0,s.k6)((()=>t[32]||(t[32]=[(0,s.eW)("View")]))),_:1,__:[32]})])])]),(0,s.Lk)("tr",null,[t[34]||(t[34]=(0,s.Lk)("td",null,[(0,s.Lk)("p",{class:"citation-text"},[(0,s.eW)(" Siaperas R, Taxeidis G, Gioti A, Nikolaivits E, Topakas E. (2025). Multi-omics insights into the response of Aspergillus parasiticus to long-chain alkanes in relation to polyethylene modification. "),(0,s.Lk)("em",null,"Environmental Pollution"),(0,s.eW)(", 376:126386. doi: 10.1016/j.envpol.2025.126386. PMID: 40345371. ")])],-1)),(0,s.Lk)("td",R,[(0,s.Lk)("a",E,[(0,s.bF)(a,{type:"primary",size:"small"},{default:(0,s.k6)((()=>t[33]||(t[33]=[(0,s.eW)("View")]))),_:1,__:[33]})])])]),(0,s.Lk)("tr",null,[t[36]||(t[36]=(0,s.Lk)("td",null,[(0,s.Lk)("p",{class:"citation-text"},[(0,s.eW)(" Taxeidis G, Siaperas R, Foka K, Ponjavic M, Nikodinovic-Runic J, Zerva A, Topakas E. (2025). Elucidating the enzymatic response of the white rot basidiomycete Abortiporus biennis for the downgrade of polystyrene. "),(0,s.Lk)("em",null,"Environmental Pollution"),(0,s.eW)(", 374:126214. doi: 10.1016/j.envpol.2025.126214. PMID: 40189091. ")])],-1)),(0,s.Lk)("td",V,[(0,s.Lk)("a",j,[(0,s.bF)(a,{type:"primary",size:"small"},{default:(0,s.k6)((()=>t[35]||(t[35]=[(0,s.eW)("View")]))),_:1,__:[35]})])])]),(0,s.Lk)("tr",null,[(0,s.Lk)("td",null,[(0,s.Lk)("p",N,[t[38]||(t[38]=(0,s.eW)(" Arecco N, Mocavini I, Blanco E, Ballaré C, Libman E, Bonnal S, Irimia M, Di Croce L. (2024). Alternative splicing decouples local from global PRC2 activity. ")),t[39]||(t[39]=(0,s.Lk)("em",null,"Molecular Cell",-1)),t[40]||(t[40]=(0,s.eW)(", 84(6):1049-1061.e8. doi: 10.1016/j.molcel.2024.02.011. PMID: 38452766 ")),(0,s.bF)(o,{content:"This manuscript uses the proteomicslfq pipeline, an early version of quantms",placement:"top",effect:"light"},{default:(0,s.k6)((()=>t[37]||(t[37]=[(0,s.Lk)("span",{class:"tooltip-trigger"},"*",-1)]))),_:1,__:[37]}),t[41]||(t[41]=(0,s.eW)(" . "))])]),(0,s.Lk)("td",K,[(0,s.Lk)("a",H,[(0,s.bF)(a,{type:"primary",size:"small"},{default:(0,s.k6)((()=>t[42]||(t[42]=[(0,s.eW)("View")]))),_:1,__:[42]})])])]),(0,s.Lk)("tr",null,[t[44]||(t[44]=(0,s.Lk)("td",null,[(0,s.Lk)("p",{class:"citation-text"},[(0,s.eW)(" Truong P, The M, Käll L. (2023). Triqler for Protein Summarization of Data from Data-Independent Acquisition Mass Spectrometry. "),(0,s.Lk)("em",null,"J Proteome Res"),(0,s.eW)(", 22(4):1359-1366. doi: 10.1021/acs.jproteome.2c00607. PMID: 36988210; PMCID: PMC10088044. ")])],-1)),(0,s.Lk)("td",O,[(0,s.Lk)("a",Y,[(0,s.bF)(a,{type:"primary",size:"small"},{default:(0,s.k6)((()=>t[43]||(t[43]=[(0,s.eW)("View")]))),_:1,__:[43]})])])])])])])])),_:1})])),_:1})])]),t[48]||(t[48]=(0,s.Fv)('<section class="publications-section" data-v-e2359424><div class="publication-category container" data-v-e2359424><h2 class="category-title" data-v-e2359424>How to Cite quantms</h2><div class="citation-guidelines" data-v-e2359424><p class="citation-intro" data-v-e2359424> If you use quantms or the data generated within quantms.org, please cite the main publication: </p><div class="citation-example" data-v-e2359424><div class="example-content" data-v-e2359424><p class="citation-text-example" data-v-e2359424> Dai C, Pfeuffer J, Wang H, Zheng P, Käll L, Sachsenberg T, Demichev V, Bai M, Kohlbacher O, Perez-Riverol Y. (2024). quantms: a cloud-based pipeline for quantitative proteomics enables the reanalysis of public proteomics data. <em data-v-e2359424>Nature Methods</em>, 21(9):1603-1607. doi: <a href="https://doi.org/10.1038/s41592-024-02343-1" target="_blank" data-v-e2359424>10.1038/s41592-024-02343-1</a>. </p></div></div><div class="citation-note" data-v-e2359424><p data-v-e2359424> If you use any of the quantms components, please cite the relevant publication from our list of team publications. </p></div></div></div></section>',1))])}}};var Q=a(6262);const Z=(0,Q.A)(G,[["__scopeId","data-v-e2359424"]]),U=Z},9999:(e,t,a)=>{a(6422)}}]);
//# sourceMappingURL=557.7269570e.js.map