"use strict";(self["webpackChunkquantms_org"]=self["webpackChunkquantms_org"]||[]).push([[785],{5785:(e,a,t)=>{t.r(a),t.d(a,{default:()=>E});var i=t(6796),l=(t(6422),t(1857)),n=(t(9303),t(641));const s={class:"publications-page"},o={class:"publications-section"},r={class:"publication-category container"},c={class:"publication-card"},u={class:"publication-links"},k={href:"https://doi.org/10.1038/s41592-024-02343-1",target:"_blank",class:"publication-link"},p={class:"publication-category container"},d={class:"publications-table-container"},m={class:"publications-table"},b={class:"link-column"},h={href:"https://doi.org/10.1016/j.jprot.2025.105440",target:"_blank",class:"publication-link"},L={class:"link-column"},f={href:"https://doi.org/10.1038/s41592-024-02197-7",target:"_blank",class:"publication-link"},g={class:"link-column"},v={href:"https://doi.org/10.1002/pmic.202300188",target:"_blank",class:"publication-link"},P={class:"link-column"},M={href:"https://doi.org/10.1021/acs.jproteome.2c00812",target:"_blank",class:"publication-link"},y={class:"link-column"},_={href:"https://doi.org/10.1093/bioinformatics/btab838",target:"_blank",class:"publication-link"},W={class:"link-column"},D={href:"https://doi.org/10.1038/s41467-021-26111-3",target:"_blank",class:"publication-link"},z={class:"link-column"},S={href:"https://doi.org/10.1101/2024.05.24.595489",target:"_blank",class:"publication-link"},C={class:"publications-table-container"},J={class:"publications-table"},F={class:"link-column"},I={href:"https://doi.org/10.1021/acs.jproteome.2c00607",target:"_blank",class:"publication-link"},w={class:"link-column"},A={href:"https://doi.org/10.1016/j.envpol.2025.126386",target:"_blank",class:"publication-link"},B={class:"link-column"},T={href:"https://doi.org/10.1016/j.envpol.2025.126214",target:"_blank",class:"publication-link"},x={__name:"index",setup(e){return(e,a)=>{const t=l.S2,x=i.v$,R=i.q;return(0,n.uX)(),(0,n.CE)("div",s,[(0,n.Lk)("section",o,[(0,n.Lk)("div",r,[a[5]||(a[5]=(0,n.Lk)("h2",{class:"category-title"},"Main Publication",-1)),(0,n.Lk)("div",c,[a[1]||(a[1]=(0,n.Lk)("h3",{class:"publication-title"}," quantms: a cloud-based pipeline for quantitative proteomics enables the reanalysis of public proteomics data ",-1)),a[2]||(a[2]=(0,n.Lk)("p",{class:"publication-authors"}," Dai C, Pfeuffer J, Wang H, Zheng P, Käll L, Sachsenberg T, Demichev V, Bai M, Kohlbacher O, Perez-Riverol Y. ",-1)),a[3]||(a[3]=(0,n.Lk)("p",{class:"publication-journal"}," Nature Methods. 2024 Sep;21(9):1603-1607. ",-1)),a[4]||(a[4]=(0,n.Lk)("p",{class:"publication-details"}," doi: 10.1038/s41592-024-02343-1. Epub 2024 Jul 4. PMID: 38965444; PMCID: PMC11399091. ",-1)),(0,n.Lk)("div",u,[(0,n.Lk)("a",k,[(0,n.bF)(t,{type:"primary",size:"small"},{default:(0,n.k6)((()=>a[0]||(a[0]=[(0,n.eW)("View Publication")]))),_:1,__:[0]})])])])]),(0,n.Lk)("div",p,[a[28]||(a[28]=(0,n.Lk)("h2",{class:"category-title"},"Publications",-1)),(0,n.bF)(R,{type:"border-card",class:"publication-tabs"},{default:(0,n.k6)((()=>[(0,n.bF)(x,{label:"Team Publications (7)"},{default:(0,n.k6)((()=>[(0,n.Lk)("div",d,[(0,n.Lk)("table",m,[a[20]||(a[20]=(0,n.Lk)("thead",null,[(0,n.Lk)("tr",null,[(0,n.Lk)("th",null,"Citation"),(0,n.Lk)("th",{class:"link-column"},"Link")])],-1)),(0,n.Lk)("tbody",null,[(0,n.Lk)("tr",null,[a[7]||(a[7]=(0,n.Lk)("td",null,[(0,n.Lk)("p",{class:"citation-text"},[(0,n.eW)(" Zheng P, Audain E, Webel H, Dai C, Klein J, Hitz MP, Sachsenberg T, Bai M, Perez-Riverol Y. (2025). Ibaqpy: A scalable Python package for baseline quantification in proteomics leveraging SDRF metadata. "),(0,n.Lk)("em",null,"Journal of Proteomics"),(0,n.eW)(", 317:105440. doi: 10.1016/j.jprot.2025.105440. PMID: 40268243. ")])],-1)),(0,n.Lk)("td",b,[(0,n.Lk)("a",h,[(0,n.bF)(t,{type:"primary",size:"small"},{default:(0,n.k6)((()=>a[6]||(a[6]=[(0,n.eW)("View")]))),_:1,__:[6]})])])]),(0,n.Lk)("tr",null,[a[9]||(a[9]=(0,n.Lk)("td",null,[(0,n.Lk)("p",{class:"citation-text"},[(0,n.eW)(" Pfeuffer J, Bielow C, Wein S, Jeong K, Netz E, Walter A, Alka O, Nilse L, Colaianni PD, McCloskey D, Kim J, Rosenberger G, Bichmann L, Walzer M, Veit J, Boudaud B, Bernt M, Patikas N, Pilz M, Startek MP, Kutuzova S, Heumos L, Charkow J, Sing JC, Feroz A, Siraj A, Weisser H, Dijkstra TMH, Perez-Riverol Y, Röst H, Kohlbacher O, Sachsenberg T. (2024). OpenMS 3 enables reproducible analysis of large-scale mass spectrometry data. "),(0,n.Lk)("em",null,"Nature Methods"),(0,n.eW)(", 21(3):365-367. doi: 10.1038/s41592-024-02197-7. PMID: 38366242. ")])],-1)),(0,n.Lk)("td",L,[(0,n.Lk)("a",f,[(0,n.bF)(t,{type:"primary",size:"small"},{default:(0,n.k6)((()=>a[8]||(a[8]=[(0,n.eW)("View")]))),_:1,__:[8]})])])]),(0,n.Lk)("tr",null,[a[11]||(a[11]=(0,n.Lk)("td",null,[(0,n.Lk)("p",{class:"citation-text"},[(0,n.eW)(" Wang H, Dai C, Pfeuffer J, Sachsenberg T, Sanchez A, Bai M, Perez-Riverol Y. (2023). Tissue-based absolute quantification using large-scale TMT and LFQ experiments. "),(0,n.Lk)("em",null,"Proteomics"),(0,n.eW)(", 23(20):e2300188. doi: 10.1002/pmic.202300188. PMID: 37488995. ")])],-1)),(0,n.Lk)("td",g,[(0,n.Lk)("a",v,[(0,n.bF)(t,{type:"primary",size:"small"},{default:(0,n.k6)((()=>a[10]||(a[10]=[(0,n.eW)("View")]))),_:1,__:[10]})])])]),(0,n.Lk)("tr",null,[a[13]||(a[13]=(0,n.Lk)("td",null,[(0,n.Lk)("p",{class:"citation-text"},[(0,n.eW)(" Bai M, Deng J, Dai C, Pfeuffer J, Sachsenberg T, Perez-Riverol Y. (2023). LFQ-Based Peptide and Protein Intensity Differential Expression Analysis. "),(0,n.Lk)("em",null,"Journal of Proteome Research"),(0,n.eW)(", 22(6):2114-2123. doi: 10.1021/acs.jproteome.2c00812. PMID: 37220883; PMCID: PMC10243145. ")])],-1)),(0,n.Lk)("td",P,[(0,n.Lk)("a",M,[(0,n.bF)(t,{type:"primary",size:"small"},{default:(0,n.k6)((()=>a[12]||(a[12]=[(0,n.eW)("View")]))),_:1,__:[12]})])])]),(0,n.Lk)("tr",null,[a[15]||(a[15]=(0,n.Lk)("td",null,[(0,n.Lk)("p",{class:"citation-text"},[(0,n.eW)(" Umer HM, Audain E, Zhu Y, Pfeuffer J, Sachsenberg T, Lehtiö J, Branca RM, Perez-Riverol Y. (2022). Generation of ENSEMBL-based proteogenomics databases boosts the identification of non-canonical peptides. "),(0,n.Lk)("em",null,"Bioinformatics"),(0,n.eW)(", 38(5):1470-1472. doi: 10.1093/bioinformatics/btab838. PMID: 34904638; PMCID: PMC8825679. ")])],-1)),(0,n.Lk)("td",y,[(0,n.Lk)("a",_,[(0,n.bF)(t,{type:"primary",size:"small"},{default:(0,n.k6)((()=>a[14]||(a[14]=[(0,n.eW)("View")]))),_:1,__:[14]})])])]),(0,n.Lk)("tr",null,[a[17]||(a[17]=(0,n.Lk)("td",null,[(0,n.Lk)("p",{class:"citation-text"},[(0,n.eW)(" Dai C, Füllgrabe A, Pfeuffer J, Solovyeva EM, Deng J, Moreno P, Kamatchinathan S, Kundu DJ, George N, Fexova S, Grüning B, Föll MC, Griss J, Vaudel M, Audain E, Locard-Paulet M, Turewicz M, Eisenacher M, Uszkoreit J, Van Den Bossche T, Schwämmle V, Webel H, Schulze S, Bouyssié D, Jayaram S, Duggineni VK, Samaras P, Wilhelm M, Choi M, Wang M, Kohlbacher O, Brazma A, Papatheodorou I, Bandeira N, Deutsch EW, Vizcaíno JA, Bai M, Sachsenberg T, Levitsky LI, Perez-Riverol Y. (2021). A proteomics sample metadata representation for multiomics integration and big data analysis. "),(0,n.Lk)("em",null,"Nature Communications"),(0,n.eW)(", 12(1):5854. doi: 10.1038/s41467-021-26111-3. PMID: 34615866; PMCID: PMC8494749. ")])],-1)),(0,n.Lk)("td",W,[(0,n.Lk)("a",D,[(0,n.bF)(t,{type:"primary",size:"small"},{default:(0,n.k6)((()=>a[16]||(a[16]=[(0,n.eW)("View")]))),_:1,__:[16]})])])]),(0,n.Lk)("tr",null,[a[19]||(a[19]=(0,n.Lk)("td",null,[(0,n.Lk)("p",{class:"citation-text"},[(0,n.eW)(" Wang D, Bouwmeester R, Zheng P, Dai C, Sanchez A, Shu K, Bai M, Umer HM, Perez-Riverol Y. (2024). Proteogenomics analysis of human tissues using pangenomes. "),(0,n.Lk)("em",null,"bioRxiv"),(0,n.eW)(" 2024.05.24.595489. doi: 10.1101/2024.05.24.595489. ")])],-1)),(0,n.Lk)("td",z,[(0,n.Lk)("a",S,[(0,n.bF)(t,{type:"primary",size:"small"},{default:(0,n.k6)((()=>a[18]||(a[18]=[(0,n.eW)("View")]))),_:1,__:[18]})])])])])])])])),_:1}),(0,n.bF)(x,{label:"Using quantms (3)"},{default:(0,n.k6)((()=>[(0,n.Lk)("div",C,[(0,n.Lk)("table",J,[a[27]||(a[27]=(0,n.Lk)("thead",null,[(0,n.Lk)("tr",null,[(0,n.Lk)("th",null,"Citation"),(0,n.Lk)("th",{class:"link-column"},"Link")])],-1)),(0,n.Lk)("tbody",null,[(0,n.Lk)("tr",null,[a[22]||(a[22]=(0,n.Lk)("td",null,[(0,n.Lk)("p",{class:"citation-text"},[(0,n.eW)(" Truong P, The M, Käll L. (2023). Triqler for Protein Summarization of Data from Data-Independent Acquisition Mass Spectrometry. "),(0,n.Lk)("em",null,"J Proteome Res"),(0,n.eW)(", 22(4):1359-1366. doi: 10.1021/acs.jproteome.2c00607. PMID: 36988210; PMCID: PMC10088044. ")])],-1)),(0,n.Lk)("td",F,[(0,n.Lk)("a",I,[(0,n.bF)(t,{type:"primary",size:"small"},{default:(0,n.k6)((()=>a[21]||(a[21]=[(0,n.eW)("View")]))),_:1,__:[21]})])])]),(0,n.Lk)("tr",null,[a[24]||(a[24]=(0,n.Lk)("td",null,[(0,n.Lk)("p",{class:"citation-text"},[(0,n.eW)(" Siaperas R, Taxeidis G, Gioti A, Nikolaivits E, Topakas E. (2025). Multi-omics insights into the response of Aspergillus parasiticus to long-chain alkanes in relation to polyethylene modification. "),(0,n.Lk)("em",null,"Environmental Pollution"),(0,n.eW)(", 376:126386. doi: 10.1016/j.envpol.2025.126386. PMID: 40345371. ")])],-1)),(0,n.Lk)("td",w,[(0,n.Lk)("a",A,[(0,n.bF)(t,{type:"primary",size:"small"},{default:(0,n.k6)((()=>a[23]||(a[23]=[(0,n.eW)("View")]))),_:1,__:[23]})])])]),(0,n.Lk)("tr",null,[a[26]||(a[26]=(0,n.Lk)("td",null,[(0,n.Lk)("p",{class:"citation-text"},[(0,n.eW)(" Taxeidis G, Siaperas R, Foka K, Ponjavic M, Nikodinovic-Runic J, Zerva A, Topakas E. (2025). Elucidating the enzymatic response of the white rot basidiomycete Abortiporus biennis for the downgrade of polystyrene. "),(0,n.Lk)("em",null,"Environmental Pollution"),(0,n.eW)(", 374:126214. doi: 10.1016/j.envpol.2025.126214. PMID: 40189091. ")])],-1)),(0,n.Lk)("td",B,[(0,n.Lk)("a",T,[(0,n.bF)(t,{type:"primary",size:"small"},{default:(0,n.k6)((()=>a[25]||(a[25]=[(0,n.eW)("View")]))),_:1,__:[25]})])])])])])])])),_:1})])),_:1})])]),a[29]||(a[29]=(0,n.Fv)('<section class="publications-section" data-v-16215973><div class="publication-category container" data-v-16215973><h2 class="category-title" data-v-16215973>How to Cite quantms</h2><div class="citation-guidelines" data-v-16215973><p class="citation-intro" data-v-16215973> If you use quantms or the data generated within quantms.org, please cite the main publication: </p><div class="citation-example" data-v-16215973><div class="example-content" data-v-16215973><p class="citation-text-example" data-v-16215973> Dai C, Pfeuffer J, Wang H, Zheng P, Käll L, Sachsenberg T, Demichev V, Bai M, Kohlbacher O, Perez-Riverol Y. (2024). quantms: a cloud-based pipeline for quantitative proteomics enables the reanalysis of public proteomics data. <em data-v-16215973>Nature Methods</em>, 21(9):1603-1607. doi: <a href="https://doi.org/10.1038/s41592-024-02343-1" target="_blank" data-v-16215973>10.1038/s41592-024-02343-1</a>. </p></div></div><div class="citation-note" data-v-16215973><p data-v-16215973> If you use any of the quantms components, please cite the relevant publication from our list of team publications. </p></div></div></div></section>',1))])}}};var R=t(6262);const V=(0,R.A)(x,[["__scopeId","data-v-16215973"]]),E=V},6262:(e,a)=>{a.A=(e,a)=>{const t=e.__vccOpts||e;for(const[i,l]of a)t[i]=l;return t}},6422:()=>{},9303:(e,a,t)=>{t(6422)}}]);
//# sourceMappingURL=785.d0011f26.js.map